docs/content/Workspace/Inspecting-Pre-aggregations.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Pre-aggregation inspection
+permalink: /cloud/inspecting-pre-aggregations
+category: Workspace
+menuOrder: 5
+---
+
+The Pre-Aggregations page in Cube Cloud allows you to inspect all
+pre-aggregations in your deployment at a glance. You can see which
+pre-aggregations are accelerating queries, which are not, and which are not
+being refreshed, along with the last 24 hours of build history.
+
+<Screenshot src="https://ucarecdn.com/eb10de99-4ff6-4a9f-abd5-736971497583/" />
+
+## Exploring pre-aggregations
+
+You can switch between [pre-aggregations](#pre-aggregations) and
+[build history](#build-history) by switching between the&nbsp;
+&ZeroWidthSpace;<Btn>Pre-Aggregations</Btn> and <Btn>Build History</Btn> tabs.
+
+## Pre-aggregations
```

Comment:
I don't think that docs that describe the UI are very useful; they don't add much to what's already evident at the glance. Here's what I think we can add:
* provide advice for what should be the ballpark for the partition build duration and partition size; let's ask Pavel for advice and bake this into docs (the correct way to do that will be to add to the "pre-aggs" section and then cross-link from the "workspace / pre-aggs" page, right?)